src/System.Numerics.Vectors/src/System/Numerics/Vector4_Intrinsics.cs

```diff
@@ -106,7 +106,8 @@ public void CopyTo(Single[] array)
         public void CopyTo(Single[] array, int index)
         {
             if (array == null)
-                throw new ArgumentNullException("values");
+                // Match the JIT's exception type here. For perf, a NullReference is thrown instead of an ArgumentNull.
```

Comment:
I just moved them all to have braces because I prefer that anyways, and those were the only arg checks in the file so it wasn't very much code to change.